### PR TITLE
Added OT-Dockerlinter integration

### PR DESCRIPTION
**Pull request by iamabhishek-dubey**
Signed-off-by: iamabhishek-dubey <abhishekbhardwaj510@gmail.com>

Extension of https://github.com/jenkinsci/analysis-model/pull/537

### CODE REVIEW

**Comment by iamabhishek-dubey** [comment]
Looks like it's dependent on the new release of analysis model

File: SUPPORTED-FORMATS.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ the [Analysis Parsers Library](https://github.com/jenkinsci/analysis-model/).
 | 70 | mypy | myPy() | - - | MyPy |  |
 | 71 | nag-fortran | nagFortran() | - - | NAG Fortran Compiler |  |
 | 72 | open-tasks | taskScanner() | ![Open Tasks Scanner](plugin/src/main/webapp/icons/open-tasks-24x24.png) ![Open Tasks Scanner](plugin/src/main/webapp/icons/open-tasks-48x48.png) | Open Tasks Scanner | - |
+| 73 | ot-docker-linter | otDockerLint() | ![OT Dockerlinter](plugin/src/main/webapp/icons/ot-docker-linter-48x48.png) | [OT Dockerlinter](https://github.com/opstree/OT-Dockerlinter) | |
```
Comment (uhafner) [inline SUPPORTED-FORMATS.md:93]: You need to generate that file, see comment on top on how to do that

File: plugin/src/main/webapp/icons/LICENSE.txt [hunk 2]
```diff
@@ -179,3 +179,12 @@ Taken from https://hadolint.github.io/hadolint/img/cat_container.png and rescale
 Usage consent via Github comment at https://github.com/hadolint/hadolint/issues/449#issuecomment-658594755
 
 ------------------------------------------------------------------------------
+
+------------------------------------------------------------------------------
+
+OT-Dockerlinter icons:
+
+The OT-Dockerlinter icon is referenced from https://github.com/opstree/OT-Dockerlinter/raw/main/static/ot-dockerlinter.svg
+No modification has been made only converted to PNG and resized
```
Comment (uhafner) [inline plugin/src/main/webapp/icons/LICENSE.txt:188]: Do they write anything about a license? This is somewhat problematic with images.

**Review by uhafner — CHANGES_REQUESTED** [review]
You need to bump the version of analysis-pom in the pom.xml as well.

File: plugin/src/main/webapp/icons/LICENSE.txt (hunk 2 above)
Comment (iamabhishek-dubey) [reply plugin/src/main/webapp/icons/LICENSE.txt:188]: I am the maintainer of the OT-Dockerlinter repo, so I don't think there will be any license issue

**Comment by iamabhishek-dubey** [comment]
@uhafner please review

**Comment by uhafner** [comment]
Thanks!

**Comment by iamabhishek-dubey** [comment]
Hey @uhafner ,

When this feature will be released?

**Comment by uhafner** [comment]
When I finished testing and documentation of the new release. (And fixed some missing security concerns in the new features). This is a rather big release that also changes serialization and hence needs a careful release. I wanted to release before Christmas, but I am not sure if is better to use the quite time after Christmas for the release.

**Pull request merged**